src/main/js/game/_reducers/ServerEventsReducer.js

```diff
@@ -23,12 +23,9 @@ export default class ServerEventsReducer {
       UserInterfaceUtils.unsetMessage(newState)
       break
 
-    case 'INIT_PLAYER':
-      newState.player = action.value
-      break
-
-    case 'INIT_OPPONENT':
-      newState.opponent = action.value
+    case 'INIT_PLAYER_AND_OPPONENT':
```

Comment:
Nice aggregations!

I was actually thinking to aggregate all events in one.
I stated with splitting them: UPDATE_LIBRARY_SIZE, UPDATE_HAND as I thought it was a good idea. At the beginning was good, as I was sending only a couple of these, but then I realised that was valid only for simple cards. Most of the cards change everything and need to send everything anyway.